src/main/java/oc/wh40k/units/ty/TYBroodBrothersHeavyWeapons.java

```diff
@@ -0,0 +1,48 @@
+package oc.wh40k.units.ty;
+
+
+
+import oc.BuildaVater;
+
+import oc.OptionsGruppeEintrag;
+
+import oc.RuestkammerElement;
+
+
+
+import java.util.Vector;
+
+
+
+public class TYBroodBrothersHeavyWeapons extends RuestkammerElement {
```

Comment:
Remove superclass. This class is never instantiated. It only holds a single static method.